### ISSUE TITLE
Group select via mouse drag only selects ports; not nodes.

### ISSUE THREAD

**Issue by aeonik** [rncbc/qpwgraph#29]
Hello,
I have an enhancement request to fix a minor issue with Node Selection workflow.

Issue:
- `Ctrl+LeftMouseClick`: allows selection of **multiple nodes**. After selecting, clicking and dragging these nodes will move the entire selected group.
- `LeftMouseClick+Drag`: allows for selection of **multiple ports**, but I would expect to select multiple nodes.

Suggestion: 

- When using "Mouse Drag Group Select": perhaps add a modifier to select groups of ports; then make the selection of a group nodes the default?

Example: 
- `Shift+LeftMouseClick+Drag` selects ports
- `LeftMouseClick+Drag` selects nodes

Screenshot for context:

![image](https://github.com/rncbc/qpwgraph/assets/354736/ad1c2a34-6250-4e72-ba32-e25cdcbedd0f)
![image](https://github.com/rncbc/qpwgraph/assets/354736/ef827621-e7ba-4cc3-b3b8-9485392c2455)

I believe the relevant code is located here:
https://github.com/rncbc/qpwgraph/blob/db3a51b9de64272eae46c9a72e4b1f9753f4a8cd/src/qpwgraph_canvas.cpp#L656C2-L676

Regards,
Aeonik

**Comment by rncbc** [comment]
see https://gitlab.freedesktop.org/rncbc/qpwgraph/-/issues/68

TL;DR; selecting ports takes priority; selecting multiple nodes in lasso/rubberband is possible as long ports are not hit;

hint: use boolean "and" (shift) and "xor" (ctrl) to your advantage.

**Comment by dewiweb** [comment]
Is it possible to select multiple ports and then connect all of them by drag and drop (for example, select 8 input ports then connect them directly to 8 output in one opération)?

**Comment by rncbc** [comment]
> Is it possible to select multiple ports and then connect all of them by drag and drop (for example, select 8 input ports then connect them directly to 8 output in one opération)?

nope. you either hit [Ctrl+C] / [Ins] to connect or [Ctrl+D] / [Del] to disconnect them all at once.
